pyenv-win/libexec/libs/shim.py

```diff
@@ -0,0 +1,44 @@
+import os
+import shutil
+import subprocess
+import sys
+
+if __name__ == "__main__":
+    if not getattr(sys, 'frozen', False):
+        sys.stderr.write("This should be run as a frozen exe\n")
+        sys.exit(1)
+
+    pyenv_root = os.path.dirname(os.path.dirname(sys.executable))
+
+    pyenv_vbs = os.path.join(pyenv_root, "libexec", "pyenv.vbs")
+    result = subprocess.run(["cscript", "/nologo", pyenv_vbs, "vname"], capture_output=True, text=True)
+    if result.returncode != 0:
+        sys.stderr.write(result.stdout)
+        sys.exit(result.returncode)
+
+    ver = result.stdout.strip()
```

Comment:
This can't hande the multi line response.

This is a valid value for `result.stdout`
```
3.12.2
3.11.8
3.10.11
3.9.13
3.8.10
3.7.4

```

Reply:
Because of https://github.com/pyenv-win/pyenv-win/pull/319

Reply:
Thanks, should support multiple versions now